addons/main/GPS2/fn_gps.sqf

```diff
@@ -0,0 +1,163 @@
+/*
+* Description:
+* Spieler-Anzeige auf Karte und GPS
+* 
+* Author:
+* [GNC]Lord-MDB, form
+*
+* Arguments:
+*
+* Return Value:
+*
+* Server Only:
+* No
+* 
+* Global:
+* No
+* 
+* API:
+* No
+* 
+* Example:
+* [] call FUNC(gps);
+*/
+
+#include "macros.hpp";
+
+#define MAX_MARKER_AGE 10
+
+GVAR(HashMapAlive) = createHashMap;
+GVAR(HashMapDead) = createHashMap;
+GVAR(HashMapTime) = createHashMap;
+
+// create special local player marker
+private _markerOwn = format["OPT_GPS_OWN_%1", getPLayerID player];
+GVAR(markerOwn) = createMarkerLocal [_markerOwn, position (vehicle player)];
+GVAR(markerOwn) setMarkerTypeLocal "mil_circle_noShadow";  
+GVAR(markerOwn) setMarkerColorLocal "ColorYellow";  
+GVAR(markerOwn) setMarkerSizeLocal [0.7, 0.7];
+GVAR(markerOwn) setMarkerAlphaLocal 1;
+
+[{
+    // update own player marker (circle only)
+    GVAR(markerOwn) setMarkerPosLocal (vehicle player);
+
+    // get all players from own side
+    private _unitsAlive = allUnits select {!(damage _x > 0.9) && (side group _x) isEqualTo (side group player)};
+    private _unitsDead = playableUnits select {damage _x > 0.9 && (side group _x) isEqualTo (side group player)};
+
+    // update alive players
+    {
+        private _name = name _x;
+        private _id = getPlayerID _x;
+        private _markerAlive = format["OPT_GPS_ALIVE_%1", _id];
+        private _markerDead = format["OPT_GPS_DEAD_%1", _id];
+
+        // Zeit des letzten Updates speichern (für späteren Cleanup)
+        GVAR(HashMapTime) set [_id, time];
+
+        // Spielermarker nur einmalig initialisieren
+        if !(_id in GVAR(HashMapAlive)) then
+        {
+            createMarkerLocal [_markerAlive, (vehicle _x)];
+            _markerAlive setMarkerAlphaLocal 0;
+            _markerAlive setMarkerTypeLocal "MemoryFragment"; // brauchbare Spielermarker: MemoryFragment, mil_triangle_noShadow, mil_start_noShadow, mil_arrow_noShadow
+            _markerAlive setMarkerSizeLocal [0.5, 0.5];
+            _markerAlive setMarkerColor "ColorWhite";
+            GVAR(HashMapAlive) set [_id, _markerAlive];
+        };
+
+        _markerAlive setMarkerPosLocal (vehicle _x);
+        _markerAlive setMarkerDirLocal getDirVisual (vehicle _x) + 90; // Um 90 Grad drehen damit die Ausrichtung vom "MemoryFragment" Icon zur Blickrichtung passt
+        _markerAlive setMarkerAlphaLocal 0.7;
+        _markerDead setMarkerAlphaLocal 0;
+
+        // keinen Spielernamen beim eigenen Marker oder wenn es deaktiviert ist anhängen
+        if (!(GVAR(SHOW_PLAYERNAMES)) || _x isEqualTo player) then
+        {
+            _markerAlive setMarkerTextLocal "";
+            continue;
```

Comment:
Ist ds zulässig ijn Clib mit continue?

Reply:
Ist ein normaler SQF Befehl.

Reply:
ja das weiß ich, abe rnicht alle Arma befehle sind in CLIB zulässig, wie zb alle Sleep Befehls versionen.